helper/call.go

```diff
@@ -50,12 +49,12 @@ type IContractCaller interface {
 	DecodeNewHeaderBlockEvent(common.Address, *ethTypes.Receipt, uint64) (*rootchain.RootchainNewHeaderBlock, error)
 	// decode validator events
 	DecodeValidatorTopupFeesEvent(common.Address, *ethTypes.Receipt, uint64) (*stakinginfo.StakinginfoTopUpFee, error)
-	DecodeValidatorJoinEvent(sdk.AccAddress, *ethTypes.Receipt, uint64) (*stakinginfo.StakinginfoStaked, error)
-	DecodeValidatorStakeUpdateEvent(sdk.AccAddress, *ethTypes.Receipt, uint64) (*stakinginfo.StakinginfoStakeUpdate, error)
-	DecodeValidatorExitEvent(sdk.AccAddress, *ethTypes.Receipt, uint64) (*stakinginfo.StakinginfoUnstakeInit, error)
-	DecodeSignerUpdateEvent(sdk.AccAddress, *ethTypes.Receipt, uint64) (*stakinginfo.StakinginfoSignerChange, error)
+	DecodeValidatorJoinEvent(common.Address, *ethTypes.Receipt, uint64) (*stakinginfo.StakinginfoStaked, error)
```

Comment:
This should be `sdk.AccAddress`. Let's avoid `common.Address` usage

Reply:
@jdkanani is there a reason to maintain `common.Address` here?

Reply:
`common.Address` is from Bor (Geth lib). It is required for contract go files, which are generated from Geth directly. ABI to go files. Unfortunately, we won't be able to change it.

Reply:
Ignore my previous comment @anilCSE. We can use `sdk.AccAddress` here.

Reply:
I changed to use **sdk.AccAddress**